include_core/omrhookable.h

```diff
@@ -65,12 +70,35 @@ typedef struct J9HookInterface {
 #define J9HOOK_AGENTID_DEFAULT  ((uintptr_t)1)
 #define J9HOOK_AGENTID_LAST  ((uintptr_t)-1)
 
+/* time threshold (=100 milliseconds) for triggering the tracepoint  */
+#define DEFAULT_THRESHOLD_IN_MILLISECONDS_WARNNING_CALLBACK_ELAPSED_TIME	100
+
+/* for generating Hook dump section */
+#define INITIALIZE_HOOKDUMP(interface) {interface.common.eventSize = sizeof(interface.flags) / sizeof(interface.flags[0]); interface.common.infos4Dump = &interface.infos4Dump[0];}
```

Comment:
This needs to be namespaced as well.

Multi-line macros are usually embedded in `do{ }while(0)` loops.